src/app/app-routing.module.ts

```diff
@@ -1,8 +1,15 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { BookDetailsComponent } from './components/book-details/book-details.component';
 
 
-const routes: Routes = [];
+const routes: Routes = [
+    {path: '',
+        redirectTo: '/history',
+        pathMatch: 'full'},
+    {path: 'history', 
+        loadChildren: () => import('./pages/history/history.module').then(m => m.HistoryModule)},
+];
```

Comment:
Also, i guess there should be some default route.
I mean, 404 page or redirect from any page to history.